src/bapi/blueprints/discord.py

```diff
@@ -41,7 +41,7 @@ def discord_auth():
         seeker_port = int(seeker_port)
     except ValueError:
         seeker_port = ""
-    if not isinstance(seeker_port, int) or seeker_port > 65535 or seeker_port < 10000:
+    if not isinstance(seeker_port, int) or seeker_port > 65535 or seeker_port < 444:
```

Comment:
```suggestion
    if not isinstance(seeker_port, int) or seeker_port > 65535 or seeker_port < 1023:
```

The top range of the "well-known" ports is 1023. If you're trying to limit the port ranges, this would be as low as BYOND should be going.